common/persistence/src/main/resources/db/migration/R__createScanInstanceView.sql

```diff
@@ -6,6 +6,6 @@ CREATE OR REPLACE VIEW scan_instance_view AS
 			typical_attenuation,
 			minimum_attenuation,
 			seconds_since_last_scan,
-			submitted_at
-	FROM data_donation.scan_instance 
+			scan_instance.submitted_at
+	FROM data_donation.scan_instance
 		INNER JOIN data_donation.exposure_window ON exposure_window_id = exposure_window.id;
```

Comment:
please check with @DKOVACS5 if the view for the export is still necessary or should be changed, since we have now a date field available

Reply:
so it was previously the submitted_at of the exposure window correct? I currently don't see the problem there

Reply:
The new date column make this view unnecessary

Reply:
Is it alright to delete the view in this case?

Reply:
Yes and sorry for the late response